drivers/hmis_csv_twenty_twenty_two/app/models/hmis_csv_twenty_twenty_two/exporter/client.rb

```diff
@@ -30,17 +30,29 @@ def export! client_scope:, path:, export:
       )
     end
 
-    def apply_overrides row, data_source_id: # rubocop:disable Lint/UnusedMethodArgument
+    def apply_overrides(row, data_source_id:) # rubocop:disable Lint/UnusedMethodArgument
       row[:FirstName] = row[:FirstName][0...50] if row[:FirstName]
       row[:MiddleName] = row[:MiddleName][0...50] if row[:MiddleName]
       row[:LastName] = row[:LastName][0...50] if row[:LastName]
       row[:NameSuffix] = row[:NameSuffix][0...50] if row[:NameSuffix]
 
-      row[:NameDataQuality] = 99 if row[:NameDataQuality].blank?
-      row[:SSNDataQuality] = 99 if row[:SSNDataQuality].blank?
-      row[:DOBDataQuality] = 99 if row[:DOBDataQuality].blank?
+      [
+        :NameDataQuality,
+        :SSNDataQuality,
+        :DOBDataQuality,
+        :GenderNone,
+        :Female,
+        :Male,
+        :NoSingleGender,
+        :Transgender,
+        :Questioning,
+        :VeteranStatus,
+        :NativeHIPacific,
```

Comment:
The other Race columns aren't required?

Reply:
They are also required, but I think they were already.  This is a rename of an old field, so data from 2020 doesn't exist until we migrate.